### PR TITLE
I run Iowa's largest wildlife preserve, and I'm rehoming a duck here

### PR DESCRIPTION
**Pull request by MandrakeRoots**
Added !quack

🦆🦆🦆🦆🦆

### CODE REVIEW

File: watched/sounds.py [hunk 1]
```diff
@@ -99,3 +99,9 @@ def skeletons(bot, sent_by, msg_text, channel=None):
     if _use_points(sent_by):
         bot.message_mailbox.put(("audio", sent_by, 'spooky-scary-skeleton_WnTSX24(1).mp3', channel))
 
+
+@messagehandler.register("twitch", "!quack")
```
Comment (MandrakeRoots) [inline watched/sounds.py:103]: This is how you will call the duck for dinner.

File: watched/sounds.py [hunk 2]
```diff
@@ -99,3 +99,9 @@ def skeletons(bot, sent_by, msg_text, channel=None):
...
+def skeletons(bot, sent_by, msg_text, channel=None):
+    if _use_points(sent_by):
+        bot.message_mailbox.put(("audio", sent_by, 'Quack.wav', channel))
```
Comment (MandrakeRoots) [inline watched/sounds.py:106]: This is where I'm leaving the duck. The little fella will answer to any name you like.

**Review by MandrakeRoots — COMMENTED** [review]
u need to fix that one thing ok

**Comment by woodenducksdontfly** [comment]
okay :)

**Pull request merged**